src/Plugin/EnqueuePlugin/ObjectManager.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace LongRunning\Plugin\EnqueuePlugin;
```

Comment:
I think it should be moved to Tests namespace

Reply:
Yes i agree